server/controllers/ReportArticleController.js

```diff
@@ -0,0 +1,147 @@
+import db from '../models';
+import response from '../helpers/response';
+
+const { Article, Report, User } = db;
+
+/**
+ *
+ *
+ * @class ReportArticleController
+ */
+class ReportArticleController {
+  /**
+  * @descripion Allow users to report articles that violate terms and agreement
+  * @static
+  * @param {*}req Express Request Object
+  * @param {*}res Express Response object
+  * @returns {object} Json response
+  * @memberof ReportArticleController
+  */
+  static async reportArticle(req, res) {
+    try {
+      const { userId } = req.user;
+      const { complaint } = req.body;
+      const { slug } = req.params;
+
+      const getArticle = await Article.findOne({
```

Comment:
Hey Shola,
Nice work.
Perhaps you would want to destructure the result from the DB. Save you a few keystrokes later.

Reply:
Destructuring the content so you write less code
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment